src/libs/runtime-utils.js

```diff
@@ -268,11 +268,21 @@ export const getDiskAppType = disk => {
   return appType
 }
 
+export const projectHasMultipleDisks = disks => {
```

Comment:
I would add documentation to this method to clarify that it does not check the app types, and therefore it is expected that the disks passed in have already been filtered by app type. And I would suggest making the method name `workspaceHasMultipleDisks`.